### ISSUE TITLE
LoadableConditionException without using @LoadableComponent

### ISSUE THREAD

**Issue by F3Ni0** [Cognifide/bobcat#208]
Bobcat Version 1.2.1
AEM BDD archetype
 
My code snippet:

```
@PageObject
public class MyClass {
	@FindBy(css = "input[name='./limit']")
	private WebElement limitTextField;

        public void setLimit(int limit) {
            int current = Integer.parseInt(limitTextField.getAttribute("aria-valuetext"));
        }
}
```

The method `setLimit` throws a `LoadableConditionException` even tho it's not annotated with `@LoadableComponent`.

The methods `getText();` or `getAttributes()` are causing the exception, however I can perform `click()` operations without throwing an exception.

Stacktrace:
```
com.cognifide.qa.bb.loadable.exception.LoadableConditionException: Loadable conditions trace info:
com.cognifide.qa.bb.aem.touch.siteadmin.aem61.SiteadminPage -> childPageWindow ~ com.cognifide.qa.bb.aem.touch.siteadmin.common.IsLoadedCondition (Fail)
Root cause: 
org.openqa.selenium.TimeoutException: Expected condition failed: waiting for com.cognifide.qa.bb.provider.selenium.BobcatWebDriverWait$$Lambda$126/439078230@79e10fb4 (tried for 10 second(s) with 1000 
MILLISECONDS interval)
Build info: version: '3.4.0', revision: 'unknown', time: 'unknown'
System info: host: 'HIDDEN', ip: 'HIDDEN', os.name: 'Windows 7', os.arch: 'amd64', os.version: '6.1', java.version: '1.8.0_111'
Driver info: com.cognifide.qa.bb.provider.selenium.webdriver.close.ClosingAwareWebDriverWrapper
	at com.cognifide.qa.bb.loadable.hierarchy.ConditionChainRunner.manageEvaluationResult(ConditionChainRunner.java:92)
	at com.cognifide.qa.bb.loadable.hierarchy.ConditionChainRunner.evaluateCondition(ConditionChainRunner.java:84)
	at com.cognifide.qa.bb.loadable.hierarchy.ConditionChainRunner.chainCheck(ConditionChainRunner.java:57)
	at com.cognifide.qa.bb.loadable.hierarchy.WebElementInterceptor.invoke(WebElementInterceptor.java:74)
	at setLimit(MyClass.java:56)
	at com.cognifide.qa.bb.loadable.hierarchy.PageObjectInterceptor.invoke(PageObjectInterceptor.java:42)
	at com.cognifide.qa.bb.frame.FrameAspect.switchFrameAndProceed(FrameAspect.java:70)
	at com.cognifide.qa.bb.frame.FrameAspect.invoke(FrameAspect.java:58)
	at my.method.set.limit(Class.java)
	at ?.And I change limit in List component Properties to 5(author/List.feature:26)
```

What do I do wrong?

**Comment by F3Ni0** [comment]
Could someone take a look at this? I still have this issue and I don't know how to handle it.

**Comment by mmanski** [comment]
```
com.cognifide.qa.bb.loadable.exception.LoadableConditionException: Loadable conditions trace info:
com.cognifide.qa.bb.aem.touch.siteadmin.aem61.SiteadminPage -> childPageWindow ~ com.cognifide.qa.bb.aem.touch.siteadmin.common.IsLoadedCondition (Fail)
Root cause: 
org.openqa.selenium.TimeoutException: Expected condition failed: waiting for com.cognifide.qa.bb.provider.selenium.BobcatWebDriverWait$$Lambda$126/439078230@79e10fb4 (tried for 10 second(s) with 1000 
MILLISECONDS interval)
```

You are not using `@LoadableComponent` annotation explicitly, but Siteadmin page which is apparently used by you does - the error message I've pasted above says that one of siteadmin page elements is not loading properly - maybe you've got different version of AEM than 6.1 or your siteadmin page renderer has been overlayed and modified

**Comment by F3Ni0** [comment]
I am using AEM 6.1

Siteadmin page renderer might be overlayed but how is it that I can perform `click()` operations but not `getAttribute()`? Is there any available workaround to turn `@LoadableComponent` annotation for anything? `try-catch` block does not help.

**Comment by mkrzyzanowski** [comment]
Should be fixed with #266 #228 in `1.6.0`
